tests/flow/test_index_create.py

```diff
@@ -1,9 +1,19 @@
 from common import *
+from pathos.pools import ProcessPool as Pool
 
 GRAPH_ID = "index"
 redis_graph = None
 redis_con = None
 
+def issue_query(i):
```

Comment:
`i` doesn't give an idea for the variable purpose, consider renaming to `graph_id`